src/clients/cli.rs

```diff
@@ -302,6 +306,7 @@ mod tests {
         type Args = Vec<String>;
         type EnvVars = HashMap<String, String>;
         type Volumes = HashMap<String, String>;
+        type EntryPoint = void::Void;
```

Comment:
Do you mind using `std::convert::Infallible` here? From my understanding, that should at some point be aliased to the never type (`!`).

Reply:
Ok